library/VTKExtensions/Rendering/vtkF3DOpenGLGridMapper.h

```diff
@@ -31,6 +31,13 @@ class vtkF3DOpenGLGridMapper : public vtkOpenGLPolyDataMapper
   vtkSetMacro(UnitSquare, double);
   ///@}
 
+  ///@{
+  /**
+   * Set the number of minor lines per square on the grid.
+   */
+  vtkSetMacro(MinorSquares, double);
+  ///@}
```

Comment:
no need for doxygen group, there is just one method

Reply:
[;)](https://github.com/f3d-app/f3d/blame/3993e05c809b9c85790c70b5225d90ed1cacb835/library/VTKExtensions/Rendering/vtkF3DOpenGLGridMapper.h#L32)

Reply:
haha, good find :)